scripts/extract_timings.py

```diff
@@ -0,0 +1,39 @@
+#! /usr/bin/env python
+# Time-stamp: <2016-05-31 14:58:27 chrplr>
+#
+
+""" Extracts slice timing information from a dicom file.
+To be passed  as argument to slice_order for slice timing correction algorithms """
+
```

Comment:
well, it crashes further down the line:

         Executing node 34c2260517be663ec808a9998c01a803 in dir: /home/cp983411/epi-example-dicom/essai/nifti/cache_dir/nipype_mem/nipype-interfaces-spm-preprocess-SliceTiming/34c2260517be663ec808a9998c01a803
160620-09:42:23,548 workflow INFO:
         Runtime memory and threads stats unavailable
        Hardlinked /home/cp983411/epi-example-dicom/nifti/StudyID-001_SeriesNumber-000009_000001ap080059.nii -> /home/cp983411/epi-example-dicom/essai/nifti/StudyID-001_SeriesNumber-000009_000001ap080059.nii ...
/home/cp983411/code/pypreprocess/pypreprocess/nipype_preproc_spm_utils.py:416: UserWarning: subject_data.anat=None (empty); skipping coregistration!
  subject_data.anat))
Traceback (most recent call last):
  File "/home/cp983411/code/pypreprocess/pypreprocess.py", line 23, in <module>
    subjects = do_subjects_preproc(sys.argv[1])
  File "/home/cp983411/code/pypreprocess/pypreprocess/nipype_preproc_spm_utils.py", line 1714, in do_subjects_preproc
    subject_data, **preproc_params) for subject_data in subjects)
  File "/usr/lib/python2.7/dist-packages/joblib/parallel.py", line 800, in __call__
    while self.dispatch_one_batch(iterator):
  File "/usr/lib/python2.7/dist-packages/joblib/parallel.py", line 658, in dispatch_one_batch
    self._dispatch(tasks)
  File "/usr/lib/python2.7/dist-packages/joblib/parallel.py", line 566, in _dispatch
    job = ImmediateComputeBatch(batch)
  File "/usr/lib/python2.7/dist-packages/joblib/parallel.py", line 180, in __init__
    self.results = batch()
  File "/usr/lib/python2.7/dist-packages/joblib/parallel.py", line 72, in __call__
    return [func(*args, **kwargs) for func, args, kwargs in self.items]
  File "/home/cp983411/code/pypreprocess/pypreprocess/nipype_preproc_spm_utils.py", line 1373, in do_subject_preproc
    subject_data.hardlink_output_files(final=True)
  File "/home/cp983411/code/pypreprocess/pypreprocess/subject_data.py", line 451, in hardlink_output_files
    setattr(self, item, linked_filename)
UnboundLocalError: local variable 'linked_filename' referenced before assignment

But this maybe because it is an unsual use case (?). For the *.ini:

#% 
disable_realign = True
disable_coregister = True
disable_segment = True
disable_normalize = True
disable_smooth = True

output_dir = essai
report = False
plot_tsdiffana = False